.ci/acceptance-tests/inspec-integration.sh

```diff
@@ -24,11 +24,6 @@ apt-get update && apt-get install google-cloud-sdk -y
 
```

Comment:
A lot of this file isn't quite right - downloading terraform, for instance, belongs in the container build process.  So does installing `unzip`, so does installing the google cloud sdk, etc etc.